preload.js

```diff
@@ -328,6 +328,8 @@ window.LokiPublicChatAPI = require('./js/modules/loki_public_chat_api');
 
 window.LokiRssAPI = require('./js/modules/loki_rss_api');
 
+window.LokiMixpanelAPI = require('./js/modules/loki_mixpanel.js');
```

Comment:
You can actually instantiate `window.mixpanel` just the one time here.
`preload.js` gets executed before the front-end kicks in, so if you do
```js
const LokiMixpanelAPI = require('./js/modules/loki_mixpanel.js');
window.mixpanel = new LokiMixPanelAPI();
```
here, you should be able to use it everywhere else (even in modules! the first comments tells the linter what global variable we are going to access)

Reply:
addressed